inference-engine/src/cldnn_engine/ops/gather.cpp

```diff
@@ -143,4 +143,50 @@ void CreateGatherOp(Program& p, const std::shared_ptr<ngraph::op::v7::Gather>& o
 }
 
 REGISTER_FACTORY_IMPL(v7, Gather);
+
+void CreateGatherOp(Program& p, const std::shared_ptr<ngraph::op::v8::Gather>& op) {
+    p.ValidateInputs(op, {2, 3, 4});
```

Comment:
The function below is a copy-paste of function for v7::Gather, so please reuse the code.